packages/cardpay-cli/scheduled-payment/cancel.ts

```diff
@@ -0,0 +1,136 @@
+import { Argv } from 'yargs';
+import { getConstant, getSDK } from '@cardstack/cardpay-sdk';
+import { getEthereumClients, getConnectionType, NETWORK_OPTION_ANY } from '../utils';
+import { Arguments, CommandModule } from 'yargs';
+
+export default {
+  command:
+    'cancel <safeAddress> <moduleAddress> <tokenAddress> <amount> <payeeAddress> <fixedUSDFee> <percentageFee> <executionGas> <maxGasPrice> <gasTokenAddress> <salt> <payAt>',
```

Comment:
I think this is too many params to define a single scheduled payment. Would be simpler to just use a single param -  `spHash`

Reply:
updated